UDV-Core/src/Modules/ConsultDoc/Readme.md

```diff
@@ -1,12 +1,6 @@
-# List of configuration files.
-The configurations files are stored in "/examples" because they are used for the demo
+### ConsultDoc
```

Comment:
Same remark as for Contribute

Reply:
please give more info about what we need in "user doc" and "developer doc"

Reply:
I will work on that and submit it in a future PR